pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationSingleValueQueriesTest.java

```diff
@@ -412,6 +412,186 @@ public void testPercentileEst99() {
         new String[]{"2146232405", "999309554"});
   }
 
+  @Test
+  public void testPercentileRawEst50() {
+    ExpectedQueryResult expectedQueryResultsBasic =
+        new ExpectedQueryResult(120000L, 0L, 240000L, 120000L, new String[]{"1107310944", "1082130431"});
```

Comment:
These are probably brittle tests since we are hardcoding the expected results? We should probably test that results are within a certain error margin of the accurate percentile?

Reply:
Makes sense. Updated.